README.md

```diff
@@ -9,11 +9,52 @@ Install
 
     $ git clone https://github.com/api-platform/demo.git
     $ cd demo
-    $ docker-compose up
+    $ docker-compose build
+    $ docker-compose up -d
 
-And go to https://localhost.
+Then load the demo fixtures:
 
-Loading Fixtures
-================
+    $ docker-compose exec php composer run load-fixtures
 
-    $ docker-compose exec php bin/console hautelook:fixtures:load --no-interaction
+You can now go to https://localhost
+
+What's included ? 
+=================
+
+This demo application contains several things you may be interested in.   
+
+Tests
+-----
+
+All entities used in this project are fully tested. Each test class extends the
+`ApiTestCase` which contains specific "API" assertions. It will make your tests
+much more straightforward than using the standard `WebTestCase` provided by Symfony. 
+
+* [Tests documentation](https://api-platform.com/docs/core/testing/)
+* [Code in test/](src/tests)
+
+Custom data provider
+--------------------
+
+This is an example where a CSV file is exposed like a standard API Platform endpoint.
+It also shows how to make this endpoint paginated.
+
+* [Data providers documentation](https://api-platform.com/docs/core/data-providers/)
+* [Code in src/DataProvider](src/DataProvider)
+
+Contributing 
+============
+
+Seen something that is wrong? A bug? Something that could be improved? Every contributions
+are welcome. 
+
+Prepare the test environment:
+
+    $ docker-compose exec php composer run prepare-test-env
+
+Then run the tests:
+
+    $ docker-compose exec php composer run tests
+
+If tests are green (you may see some deprecations warnings), you are ready to contribute!
```

Comment:
Please create a [.github/CONTRIBUTING.md](https://github.com/api-platform/core/blob/master/CONTRIBUTING.md) file instead, as it'll be detected by GitHub. This file must contain the contributions rules